wren-ai-service/eval/prediction.py

```diff
@@ -237,12 +105,18 @@ def parse_args() -> Tuple[str]:
         type=str,
         help="Eval dataset file name in the eval/dataset folder",
     )
+    parser.add_argument(
```

Comment:
we can support options for users to easily understand what are supported.

https://docs.python.org/3/library/argparse.html#choices

Reply:
nice advice! will put in the commits.